bin/mochi.js

```diff
@@ -0,0 +1,210 @@
+const shell = require("shelljs");
```

Comment:
Do you not flow type your bin scripts? I really like treating all my code as first class citizens.